From 782f63d8f858b1c14df38aaf623438d7ea2f75e1 Mon Sep 17 00:00:00 2001
From: mihirlad55 <mihirlad55@gmail.com>
Date: Mon, 10 Aug 2020 01:39:35 +0000
Subject: [PATCH] Add support for managing external status bars

This patch allows dwm to manage other status bars such as
polybar/lemonbar without them needing to set override-redirect.

The patch is developed at https://github.com/mihirlad55/dwm-anybar
---
 config.def.h |  3 ++
 dwm.c        | 14 +++++++-------
 2 files changed, 10 insertions(+), 7 deletions(-)

diff --git a/config.def.h b/config.def.h
--- a/config.def.h
+++ b/config.def.h
@@ -1,3 +1,6 @@
 static const int topbar = 1;
+static const int usealtbar = 1;
+static const char *altbarclass = "Polybar";
+static const char *altbarcmd = "$HOME/bar.sh";
